x-pack/plugins/apm/public/components/app/service_overview/service_overview_transactions_table/index.tsx

```diff
@@ -0,0 +1,329 @@
+
+  const [tableOptions, setTableOptions] = useState<{
+    pageIndex: number;
+    sort: {
+      direction: SortDirection;
+      field: SortField;
+    };
+  }>({
+    pageIndex: 0,
+    sort: DEFAULT_SORT,
+  });
+
+  const {
+    data = {
+      totalItemCount: 0,
+      items: [],
+      tableOptions: {
+        pageIndex: 0,
+        sort: DEFAULT_SORT,
+      },
+    },
+    status,
+  } = useFetcher(() => {
+    if (!start || !end) {
+      return;
+    }
+
+    return callApmApi({
+      pathname: '/api/apm/services/{serviceName}/overview_transaction_groups',
+      params: {
+        path: { serviceName },
+        query: {
+          start,
+          end,
+          uiFilters: JSON.stringify(uiFilters),
+          size: PAGE_SIZE,
+          numBuckets: 20,
+          pageIndex: tableOptions.pageIndex,
+          sortField: tableOptions.sort.field,
+          sortDirection: tableOptions.sort.direction,
+        },
+      },
+    }).then((response) => {
+      return {
+        items: response.transaction_groups,
+        totalItemCount: response.total_transaction_groups,
+        tableOptions: {
+          pageIndex: tableOptions.pageIndex,
+          sort: {
+            field: tableOptions.sort.field,
+            direction: tableOptions.sort.direction,
+          },
+        },
+      };
+    });
+  }, [
+    serviceName,
+    start,
+    end,
+    uiFilters,
+    tableOptions.pageIndex,
+    tableOptions.sort.field,
+    tableOptions.sort.direction,
+  ]);
+
+  const {
+    items,
+    totalItemCount,
+    tableOptions: { pageIndex, sort },
+  } = data;
+
+  const columns: Array<EuiBasicTableColumn<ServiceTransactionGroupItem>> = [
+    {
+      field: 'name',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnName',
+        {
+          defaultMessage: 'Name',
+        }
+      ),
+      render: (_, { name, transaction_type: transactionType }) => {
+        return (
+          <TransactionGroupLinkWrapper>
+            <EuiToolTip delay="long" content={name}>
+              <StyledTransactionDetailLink
+                serviceName={serviceName}
+                transactionName={name}
+                transactionType={transactionType}
+              >
+                {name}
+              </StyledTransactionDetailLink>
+            </EuiToolTip>
+          </TransactionGroupLinkWrapper>
+        );
+      },
+    },
+    {
+      field: 'latency',
+      name: i18n.translate(
+        'xpack.apm.serviceOverview.transactionsTableColumnLatency',
+        {
+          defaultMessage: 'Latency',
+        }
+      ),
+      width: px(unit * 10),
+      render: (_, { latency }) => {
+        return (
+          <SparkPlotWithValueLabel
+            color="euiColorVis1"
+            compact
+            series={latency.timeseries ?? undefined}
+            valueLabel={
+              latency.value !== null
+                ? asDuration(latency.value)
+                : NOT_AVAILABLE_LABEL
```

Comment:
`asDuration` already defaults to `NOT_AVAILABLE_LABEL` when there's no value

```suggestion
              asDuration(latency.value)
```